### PR TITLE
Crm 6084 - mark delete

### PR DESCRIPTION
**Pull request by nissamai**
Adds functionality to mark users in CRM when deleted.

https://app.shortcut.com/sefariatest-1/story/6084/

### CODE REVIEW

File: sefaria/helper/crm/salesforce.py [hunk 1]
```diff
@@ -110,7 +110,9 @@ def mark_as_spam_in_crm(self, uid):
         except:
             # log
             return False
-        return res
+
```
Comment (shaneerosen) [inline sefaria/helper/crm/salesforce.py:113]: Do we want to add the logging here?

**Review by shaneerosen — COMMENTED** [review]
How did you test this code?

File: sefaria/utils/user.py [hunk 2]
```diff
@@ -18,6 +22,13 @@ def delete_user_account(uid, confirm=True):
             print("Canceled.")
             return
 
+    try:
```
[the remark this answers is not in the document]
Comment (nissamai) [reply sefaria/utils/user.py:25]: I *think* that return will only get hit if the user doesn't type in "DELETE" when confirm==True? (the lines with the if block somehow get cut out of the github changes view)

[the remark this answers is not in the document]
Comment (shaneerosen) [reply sefaria/utils/user.py:25]: yes! git misled me! I was surprised, now it makes sense. sorry!

File: sefaria/helper/crm/salesforce.py (hunk 1 above)
Comment (shaneerosen) [reply sefaria/helper/crm/salesforce.py:113]: no. I actually like it better in user.py, but then let's delete that comment as well. and now that line 28 &30 are actually running things make more sense.

Comment (nissamai) [reply sefaria/helper/crm/salesforce.py:107]: Ok, removed all the log comments! Thanks shanee!

**Review by shaneerosen — APPROVED** [review]

**Pull request merged**